src/nunit-gui/Views/SettingsPages/GuiSettingsPage.cs

```diff
@@ -28,11 +28,11 @@
 using System.Data;
 using System.Text;
 using System.Windows.Forms;
+using NUnit.Gui.Model;
```

Comment:
No, the standard says "permissible but not required" I ended up with Model in one place and Model.Settings in another and it looked funny to me so I moved them both to the outside.